MPF.Processors/Redumper.cs

```diff
@@ -452,8 +452,11 @@ internal override List<OutputFile> GetOutputFiles(string baseFilename)
                             "log"),
                         new CustomOutputFile($"{baseFilename}.log", OutputFileFlags.Required,
                             DatfileExists),
-                        new([$"{baseFilename}.manufacturer", $"{baseFilename}.1.manufacturer"], OutputFileFlags.Required
+                        new([$"{baseFilename}.manufacturer", $"{baseFilename}.0.manufacturer"], OutputFileFlags.Required
                             | OutputFileFlags.Binary
+                            | OutputFileFlags.Zippable,
+                            "manufacturer_0"),
+                        new($"{baseFilename}.1.manufacturer", OutputFileFlags.Binary
                             | OutputFileFlags.Zippable,
                             "manufacturer_1"),
                         new($"{baseFilename}.2.manufacturer", OutputFileFlags.Binary
```

Comment:
If the numbering has changed to be consistent with `0, 1` instead of `1, 2`, then this can be removed.

Reply:
`.2.physical` can be removed as well for DVD.

Reply:
`.3.physical` can be added for Blu-ray.